assets/css/app.scss

```diff
@@ -152,3 +152,12 @@
 #content {
   min-height: 87vh;
 }
+
+.fa.disabled,
+.fa[disabled],
+.disabled > .fa,
```

Comment:
If we implement the button suggestions I have below then we wont need to style the icons since they will be part of the button

Reply:
removed